editor/components/block-toolbar/style.scss

```diff
@@ -18,7 +18,6 @@
 		border-right: 1px solid $light-gray-500;
 	}
 
-	// this should probably have its own class
```

Comment:
Is this actually no longer the case? I'd still agree with the comment, I don't like seeing selectors like `> div`; they seem sloppy.

Reply:
My thinking in removing this comment was perhaps fast and loose. I think it's still the case that it should be a class, but I this not happening in the near future, and instead of leaving a "note to self" might as well remove it. I remember trying to add the class, but not finding a clean way to do it.

Reply:
All of that sounds useful to know, and rather like the comment should be expanded upon rather than deleted 😄